smoderp2d/.config.txt

```diff
@@ -0,0 +1,3 @@
+[outputs]
+# yes or no
+extraout = no
```

Comment:
Detailed description to be added.

Reply:
see 70e2c7aaa96b36bfdb442cf3bb347d505ee447ff